src/host/outputStream.cpp

```diff
@@ -370,6 +370,17 @@ void ConhostInternalGetSet::NotifyAccessibilityChange(const til::rect& changedRe
     }
 }
 
+void ConhostInternalGetSet::EndConptyFrame()
+{
+    auto& g = ServiceLocator::LocateGlobals();
+    auto& gci = g.getConsoleInformation();
+
+    if (gci.IsInVtIoMode() && g.pRender)
+    {
+        g.pRender->TriggerFlush(false);
+    }
+}

```

Comment:
I don't think this needs to be in the `ConGetSet` interface. `AdaptDispatch` now has direct access to the renderer, so this is basically just:

    if (_pConApi->IsConsolePty())
    {
        _renderer.TriggerFlush(false);
    }